src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
```

Comment:
Right, I actually implemented it as an implicit first (hence the name) but then realized you can't override the `nextBackOffMillis` method.